src/java/org/apache/cassandra/service/accord/serializers/RecoverySerializers.java

```diff
@@ -90,11 +91,12 @@ void serializeOk(RecoverOk recoverOk, DataOutputPlus out, int version) throws IO
             CommandSerializers.ballot.serialize(recoverOk.accepted, out, version);
             serializeNullable(recoverOk.executeAt, out, version, CommandSerializers.timestamp);
             DepsSerializer.partialDeps.serialize(recoverOk.deps, out, version);
+            DepsSerializer.partialDeps.serialize(recoverOk.acceptedDeps, out, version);
```

Comment:
in many cases, I'd expect these to be identical to `deps`. Can we add a flags field, and use it to indicate `acceptDeps == deps`, and skip serialization/deserialization?